components/Footer.js

```diff
@@ -16,24 +17,28 @@ export default function Footer() {
       <div className="max-w-screen-xl mx-auto py-12 px-4 overflow-hidden sm:px-6 lg:px-8">
         <nav className="-mx-5 -my-2 flex flex-wrap justify-center">
           <div className="px-5 py-2">
-            <a href="/about" className="text-base leading-6 text-gray-500 hover:text-gray-900">
-              About
-            </a>
+          <div className="text-base leading-6 text-gray-500 hover:text-gray-900">
+            <Link href="/about" className="text-base leading-6 text-gray-500 hover:text-gray-900">
+             <a> About </a>
+            </Link>
+            </div>
           </div>
           <div className="px-5 py-2">
-            <a href="/blog" className="text-base leading-6 text-gray-500 hover:text-gray-900">
-              Blog
-            </a>
+            <div className="text-base leading-6 text-gray-500 hover:text-gray-900">
+              <Link href="/blog">
+              <a>Blog</a>
+              </Link>
+            </div>
           </div>
           <div className="px-5 py-2">
-            <a href="https://asyncapi.threadless.com" target="_blank" className="text-base leading-6 text-gray-500 hover:text-gray-900">
+            <a href="https://asyncapi.threadless.com" target="_blank" rel="noopener noreferrer nofollow" className="text-base leading-6 text-gray-500 hover:text-gray-900">
```

Comment:
Sorry man for that, but I see that in some parts nofollow should not be and I myself will add in a separate PR in places where it should be. Sorry for your unnecessary work! Could you please remove any nofollow occurrences from your PR? Thanks and sorry again! 😅

Reply:
So, basically I have to revert my last commit, which includes addition of `nofollow` property?

Reply:
Yes, sorry!

Reply:
No Problem :thumbsup: . Done with the changes.